### ISSUE TITLE
cmd window?

### ISSUE THREAD

**Issue by nobodyspecial** [Martenfur/TabletFriend#54]
Is there any way to hide the cmd window when launching an app?

**Comment by Martenfur** [comment]
What cmd window are you referring to?

**Comment by nobodyspecial** [comment]
> What cmd window are you referring to?

The cmd.exe window when opening an app like calc.exe in the sample layout. It pops up briefly before calculator opens.

**Comment by Martenfur** [comment]
Yeah, sadly, that's a thing. cmd essentially opens your terminal and orders it to execute a command you've set. I'm not aware of if this can be done any other way. In case of launching programs, a new command could be added, specifically for launching exes.

**Comment by nobodyspecial** [comment]
> In case of launching programs, a new command could be added, specifically for launching exes.

Yeah that's what I ended up doing, compiled with nircmd.exe in place of cmd.exe. Thanks for replying, love the app.
